### DISCUSSION TITLE
在哪关掉文章标题的打字效果

### DISCUSSION THREAD

**Discussion by hopeko** [BNDong/Cnblogs-Theme-SimpleMemory#263] [Q&A]
2.0版本后貌似一些之前有的配置项都没有了，像设置代码的字体，另外希望可以增加可以不要文章底部推荐和新闻的配置项。

**Comment by SMARK2022** [comment]
我感觉可能新闻那个估计博客园不想让关，所以估计这是后来把去广告的代码改掉的原因。
确实，+1我也希望可以关闭打字效果，太晃眼、加载速度还慢，不是太适合。。。
